src/layers/microscopy-viewer-layer/microscopy-viewer-layer.js

```diff
@@ -11,20 +12,23 @@ export class MicroscopyViewerLayer extends CompositeLayer {
     };
   }
 
+  // eslint-disable-next-line class-methods-use-this
   shouldUpdateState({ changeFlags }) {
     return changeFlags.somethingChanged;
```

Comment:
If we're not using `this`, are we risking cross-talk if there are multiple instances of the component?

Reply:
This is how deck.gl [implements](https://github.com/uber/deck.gl/blob/master/docs/api-reference/layer.md#shouldupdatestate) their state checking - I honestly don't know fully how it works, but iy doesn't rely on `this` in any way.  Is that something that happens?  Can you link?